### ISSUE TITLE
schema-reader-db Error: /bootstrap/../routes/app.php' (include_path='.:/usr/share/php')

### ISSUE THREAD

**Issue by gdconnect** [VemtoOrg/vemto2-issues#198]
**Error Message:**
schema-reader-db Error: Failed opening required '/var/www/html/myproject/bootstrap/../routes/app.php' (include_path='.:/usr/share/php')

**Stack:**
```
schema-reader-db Error: Failed opening required '/var/www/html/myproject/bootstrap/../routes/app.php' (include_path='.:/usr/share/php')

Stack:
schema-reader-db Error: #0 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Routing/Router.php(513): Illuminate\Routing\RouteFileRegistrar->register()
#1 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Routing/Router.php(467): Illuminate\Routing\Router->loadRoutes()
#2 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Routing/RouteRegistrar.php(194): Illuminate\Routing\Router->group()
#3 /var/www/html/myproject/bootstrap/app.php(18): Illuminate\Routing\RouteRegistrar->group()
#4 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Foundation/Configuration/ApplicationBuilder.php(232): {closure}()
#5 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): Illuminate\Foundation\Configuration\ApplicationBuilder->Illuminate\Foundation\Configuration\{closure}()
#6 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Container/Util.php(43): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#7 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(83): Illuminate\Container\Util::unwrapIfClosure()
#8 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(35): Illuminate\Container\BoundMethod::callBoundMethod()
#9 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Container/Container.php(690): Illuminate\Container\BoundMethod::call()
#10 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Foundation/Support/Providers/RouteServiceProvider.php(162): Illuminate\Container\Container->call()
#11 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Foundation/Support/Providers/RouteServiceProvider.php(59): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#12 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): Illuminate\Foundation\Support\Providers\RouteServiceProvider->Illuminate\Foundation\Support\Providers\{closure}()
#13 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Container/Util.php(43): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#14 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(83): Illuminate\Container\Util::unwrapIfClosure()
#15 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(35): Illuminate\Container\BoundMethod::callBoundMethod()
#16 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Container/Container.php(690): Illuminate\Container\BoundMethod::call()
#17 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Support/ServiceProvider.php(130): Illuminate\Container\Container->call()
#18 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(1122): Illuminate\Support\ServiceProvider->callBootedCallbacks()
#19 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(1100): Illuminate\Foundation\Application->bootProvider()
#20 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}()
#21 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(1099): array_walk()
#22 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/BootProviders.php(17): Illuminate\Foundation\Application->boot()
#23 /var/www/html/myproject/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(316): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap()
#24 phar:///tmp/.mount_Vemto-R2pFfm/resources/static/VMTTL2/index.php(39): Illuminate\Foundation\Application->bootstrapWith()
#25 phar:///tmp/.mount_Vemto-R2pFfm/resources/static/VMTTL2/common/Vemto.php(111): {closure}()
#26 phar:///tmp/.mount_Vemto-R2pFfm/resources/static/VMTTL2/index.php(20): Vemto\Vemto::execute()
#27 /tmp/.mount_Vemto-R2pFfm/resources/static/VMTTL2(15): require('...')
#28 {main}

```

**How to reproduce:**
1. Try to generate new project

**Expected behavior:**

**Actual behavior:**

**Additional context:**

---

Vemto 2.1.4
linux 5.15.0-118-generic
Locale: en-GB
App version: 2.1.4

**Comment by TiagoSilvaPereira** [comment]
Hi @gdconnect, I just fixed this on Vemto 2.1.5. 

If you want to reuse the same project, please create two empty PHP files inside the /routes folder:

- app.php
- app-api.php

Or create a new project.

**Comment by gdconnect** [comment]
Thank you
